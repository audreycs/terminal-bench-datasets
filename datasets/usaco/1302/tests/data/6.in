100

10 607931817 569432344
417204537 199478059 277560071132118594
779163299 1499508 383471899073795119
597077646 543882563 669968177286708301
734220053 426250646 504399770833887513
213371753 257972632 227510156588224103
112740497 288720105 219085553649613663
20319456 657208754 376549227413220024
393510570 588425608 535383371965733996
832575738 46054288 329038806801173716
599060284 521866928 630489756583508535

10 960746317 342586055
974868608 412285821 471533082261278387
484283021 207956467 186941459651712473
463262331 371686520 327838115404189595
346596516 685665953 512920469994020247
146112372 376480109 176184368535913099
80591669 799969357 181101344194184503
78093123 557386012 165074062942873137
83128888 530120850 123724183558298035
824418410 174993968 796735217973814046
143681074 826672177 230480066905058314

8 259791593 640337745
506326918 436957916 395286009244112932
848454682 189598850 41348456496286817
897936960 735012286 342206225148652429
749356645 862000551 691216866304682024
673217498 504991941 224676445106365117
747198402 327166779 197103942795775731
250977037 468283972 215619653577542961
818961924 515414817 532925249669371801

7 999214157 923580605
493187752 273611938 721007347975489410
46053839 463200189 467662843158102978
212391979 150788962 233256983361686412
658899584 781771523 1275196304404601099
65074999 45384565 82095366759064740
646203021 872197652 1006466548080628758
652533147 461660863 741493465249438927

9 849139117 94836572
728500898 195903802 466941538132240082
325324111 908059817 286460643034410632
945496621 206542139 746521539611204346
369983083 782794942 171003835899565307
464739136 543434392 435397580826855488
118483283 642525402 90071590966513836
521990434 447126515 468356834154549793
722759555 285365255 146355048188322690
282504655 3558903 34054644732008994

5 237196195 335636951
492409711 470953702 215703384849838409
673344113 678599245 256387053832851637
823032612 825705794 345748254953691148
743336307 71488100 196738304246274834
195490875 866245217 218057723198937667

10 627311817 506517717
635239214 730022726 611113457426127542
415257883 878357634 475266047196625679
198841540 513777777 368646890477529238
235666109 243925155 213216849202601156
130984483 932610293 417491938745403780
761984735 242703703 570630138135685575
443863442 561847107 541238145096984965
39364684 804232135 248439779346041531
408760903 892944442 466235488228094504
134929409 748785546 447138001226188127

10 694949826 691472356
123930798 381482375 324455492747767283
896145410 521952452 663653945722777125
235257302 552403265 372345638303436746
296578261 615597901 505501517896077056
828426007 628893518 951683526145033606
146769664 63733426 145764810763308703
738409944 804086902 835295650591814468
764875285 711391498 854455373656769484
606051330 579112644 556750820568933838
543715532 468658019 485905955886729366

10 172531650 834538751
447209593 687074702 445116960273161284
94845916 672070079 465749515186350031
175795582 653728616 345420940211811108
28658711 276947680 213266064694240218
880551787 499076121 178690090245274669
414485933 362300092 261440307223255380
95544652 611277435 294660294891597664
503011916 706449408 524351247841085092
900327477 476617192 343772333860257228
714025428 188231960 101969065738367955

6 794758496 246233235
425692576 946667883 482526478085097668
649475269 72670060 326935127949555312
257821755 568117346 320907154259750235
686943624 659486065 703599495576437368
68418357 551148936 132353310801147814
387841471 286754041 353133113179810215

10 973971648 444602384
464516025 722783686 517105974398786766
792646094 449545499 812784829082403598
972873888 709771675 1230007578364064097
764325931 251241374 753636004716291000
818862081 795475215 902924581409979262
421382134 55774758 399666337248615922
61265831 668981019 157969900366916323
642856475 194024659 585985980639530246
799202650 81510205 793472848106499605
364762826 30736148 340311390993041834

8 21658334 500004222
966484221 939668783 353151148591877625
758570753 466075326 159622411222619206
728793552 688953784 263689923907551838
996168982 807452639 364077956494679469
644729399 978353838 489242942539681602
276235378 292175763 124280485824822557
37078113 744532293 309545348357729854
847279720 163002065 95675473659010353

5 477363214 429378839
589034082 956989466 525250549291098887
689881835 74272190 245088892672791456
111830600 652996585 179321302732356398
220502508 843319110 394900925480296297
615523836 628854495 529365714052098079

10 818619034 94816446
117803098 317940569 70053389361987752
953554118 484717032 799143106417516740
154367278 320119883 96571380718684631
790741242 314790980 517767098891871543
910602750 437589388 591290243534413452
589858261 610338320 313636657073687801
509482219 526518454 376965553326669681
531168206 425760695 461984569187507543
662478244 286954948 308307390929500916
608524494 896741785 456609923883700643

7 328226008 978566557
251359659 637655212 672475317117937833
841823618 192357913 306816367091874560
994284373 25303565 247130452979377012
306095097 724431843 354079624672765220
657811199 60574231 178787608166380302
359005032 251966352 315968389490606470
262050073 780627592 821174475121906797

5 898878625 819917132
819759331 252112105 809941538655779973
3320690 640446341 248902399801913293
489121850 651882230 662154487429008963
323268938 537383556 243396113125501382
70448534 500201057 429624138358428777

10 724476950 903876497
405467794 124967084 317217765747656753
513252574 60704129 179400136890249412
6379101 994506594 344767456427593058
165932179 39386082 147586574310044615
687620292 446304612 536382928506901702
59117193 71175653 93125520266289749
588144849 131741325 239789213249779890
587392832 389549993 339635830429382572
433982624 729982638 949877160122585165
742251104 349142861 631706342692061593

6 964908503 429655694
372229544 536763317 285752274261946798
584359640 931294028 551820149983005624
994203873 956747740 1200735760345078006
121878995 727683177 220096231103070182
132380396 939612899 328024772667660344
304211570 441653169 223064348563443996

8 744482065 635642921
193955152 78779969 157917330006527959
108199875 147430073 156839106601539842
303736329 219122080 277528593007861945
120634745 201948363 179095254219219145
264802567 554199932 348702388085592776
597674689 681584943 677120771049748952
615922382 603745958 553908350335178760
146721273 779828099 441148000977032810

10 287226131 766055778
218477914 328749808 295527380210699415
251663599 637451875 439557424506877884
444833262 112671441 174073216283527005
674382869 135329845 243831490202491810
470074923 486257351 392553049384043718
229563050 999951064 814382632045113966
685693770 899243372 555227886284749173
582916962 232320342 318629362601464507
498224613 475654200 475493229782635924
991825497 722175126 659086935088641487

5 381474336 137141164
690992008 616197430 343988670123883264
649477856 121356525 249879565566476819
23940524 888901479 130281684589593894
439638088 32616216 158004505658738924
708345772 728079088 343759566907026071

7 74288178 407613053
617221563 258932987 19482380066957121
353137543 702414117 43546395382171408
697855830 802337262 164928759163356312
64332825 469208178 20272081831854433
669773249 789299023 319550380167584764
280470935 488091075 93741189885950157
458024319 812418682 269125717329995626

9 285636684 544256889
334349842 121028972 140388051826346415
31679727 343556808 162650071928479296
218176180 779378398 434242487107450699
749218766 787980912 634049596862939339
656262524 214498144 262042378034998708
539148253 744279735 434431186961137235
817539119 939049642 587544620109056662
599823470 34979111 180977529889449991
973911245 823907754 442809609278392451

7 92537256 662318007
96873937 108642803 53817483452090881
847338657 736219545 396211839468989348
219280356 961218523 418099284448689398
350058167 114864556 68338596066332481
800256972 270949895 159584837798152564
729836266 255694904 133676908016199970
286523432 953287919 654552630895652561

5 87966999 948082081
423168320 435927319 336695545124187632
649001115 561125607 234435810851471481
57292239 382552732 158417178846480707
854607112 427361885 340323115775094753
908884443 844490072 752987220156938697

8 989477735 174500305
699922111 200724062 671903583573979213
731441826 368295960 725141621812421639
844055537 663973891 884344591757586962
346767212 146862896 306612063911761784
575612205 198291970 489709097514668831
103419289 503911075 190114635756122340
687780069 501350194 688492399578542383
117527671 436353045 175373625754255886

5 890828813 376160862
606262210 941849872 664835033050018036
189414127 417013793 210835066101498525
21175422 615238780 137702650040107554
279918981 725743846 437349810410108944
636836325 939511169 891584294857877893

5 106610534 691820511
546075017 105434861 123262942497360485
857363107 946935196 295826031567776472
614485272 58536338 69542763594493905
391902864 627134946 457139896712705972
697528281 415707683 69548096040760193

9 932649433 183120284
776612023 796919460 657751037926105172
279181091 61041998 206045988071544953
376466131 963524870 336383517796592830
78008652 889337258 121927472903342767
884446046 81380096 565743517103152303
490378641 113631331 405236494197609858
718309247 702911875 619905169981010327
147338370 496374907 180231292551328623
939972799 988640986 841969194892335023

10 304509633 520109633
924711213 59709521 270239135480794573
376608265 446767868 330882251726619173
321333689 785204818 482334552236803113
601034373 54578277 143558101825012999
413076960 315629283 275850779634935464
868762598 372567405 369280384015842570
867111145 897053391 628453182431969074
53260698 594651276 304426435611282351
171971447 936771335 527135932490568536
46509533 566133394 296675134866432870

6 764213943 333446160
487770678 780041047 433478456490702924
942185239 234412434 426162298202165875
223261920 884722005 301406052247306934
260848355 24596125 178808200940492784
427530586 707533748 363478953836401159
469241030 80854816 240579414358981213

8 844463858 530055918
990616447 135251786 792911497697162835
301834563 914406602 415500871406549753
734300393 789946771 406263393167311126
137516337 731598792 436239955594270128
34843865 181051799 90698035389536962
667849879 143455550 590970775833913690
654095457 117342830 227604713135109769
634217928 263191805 392099933038390601

9 595835193 770829591
370133126 534149993 590316598094585526
502656729 138587379 396032533122778846
199513614 402410212 405039253147497815
533418607 534397310 725647845552799147
530799692 636897433 732971347449966862
464772901 68271241 319290006034991378
389811711 296894084 398074367450947101
225037330 768272104 660287862475063898
630288680 101726931 429681000777752700

7 697429805 287180778
647582530 455586783 408115500820795281
29149399 607893763 123746024195499416
679128311 517187562 450324533119389676
230267454 316358054 197601102270313234
593361905 812038234 614311979484735848
50486820 224253554 64176303316388992
85143435 844718722 97054045950795300

5 176780449 749497202
433778198 849673144 453655875180818410
454082267 437176981 172186008008869234
90004869 951779731 505554680355496926
880291856 940197813 194965165753207891
699700638 663477494 560267333845809593

6 676297066 508761986
768119979 92754111 484674931886899146
757537856 141391573 481597838985329907
884938887 553412879 716510534751494492
735762237 490690441 653624965902175795
240277486 402301646 366258616293391730
566610437 577312285 659404848866730021

10 970441940 789526919
263523251 562418660 649789750279113694
925065181 503310210 1201247034609309131
255225898 309493281 356085743044126731
822908650 210431722 852669254430130265
229353048 193737524 374280841634989389
519527850 220342999 592693524994381496
283537155 632361729 691989706999917786
973109124 412129083 1063076574570723399
298614853 243461896 396801959830801991
558943066 420031387 768241628150101094

10 139450410 569960754
938506948 494806133 370001747255721542
170539548 275888971 117764596505019998
808732504 962344620 439749354398466226
18409071 36486584 14105992527109180
457262259 822673312 443678522798324967
315953581 2132147 41556249877506844
286149096 565528517 216377267008677422
424586618 326595038 204928590591487282
856149956 40500377 121147960945561210
604958426 749690677 297005271939852202

8 191370899 29367752
174869950 89318195 5785502222314626
798208862 662262872 61254391851090407
902043670 559445567 35273280979411359
308492483 22854602 52713423230939780
998424426 191837611 131735471601279740
451439569 267804897 15385244594036079
147425487 510995745 42173175342487475
456614198 718878200 18583310891618210

9 850407884 517778907
513267220 978666386 833157040471560783
157472298 64322143 166440106143894801
495069210 123086079 291267958709828621
204459215 348287565 246213132177054089
842840766 757779411 516849610905806908
459025355 443425943 601261459006613006
168377398 589335776 334375664594302833
542104463 897414598 871437604767261735
17293161 123982597 68319414787880532

5 941128015 407386715
361472836 717914358 354174736640523628
339779278 861433227 534783442428936298
677605760 536327209 709459085404813635
237495408 152729551 236880321959799747
723515881 836782314 685600155638020058

10 947123625 865133920
565241609 49660829 505505442985014781
707374760 233221559 615794165516087373
406294463 135854518 489811037232367130
978150081 391376040 985066658393211883
880622832 336166275 1066406061559604989
902333589 685033000 1418048015022397998
910013984 171082423 1008557919154594751
232576507 528783510 646834302645531451
524412141 793861157 950596305956486270
745562457 425647043 780251742321778970

9 546063298 93033070
37711426 420134911 52741807991912445
886684863 628265217 326110299128535535
218158419 881739378 198372905866646424
5183582 488609698 45304440434685785
456659460 645248975 266321346790427072
648333097 633344763 264083388303857778
242831615 835532494 193543395458296809
119301222 353225474 82189617461880264
826642209 12523291 408544025304501846

8 859294129 773637813
750295321 912610724 1325886591429290556
418363280 906058978 942083259233239980
305683190 159975333 365462708047085843
376735869 550898420 684627954777034113
915078672 919834830 1252938431412693488
474480860 709100441 660248228319395661
416007522 90133894 413142210134273886
58389884 406095926 351004412819709515

10 453195204 875213700
178720881 106044487 111852021348545242
707313414 845359539 392476577196736689
646184296 145615047 411911606529642321
409811523 339588197 161769402939243564
368172449 216142416 142719676582401813
648100069 169805744 145392263550616354
457980522 576371587 618243090863647901
586863589 103380745 303972317437373757
54185297 532236218 287033325674751796
324213829 112142577 55110503029192773

9 539071318 759309308
840849726 483815031 779515171131739689
478723454 921279661 661361043452417958
373583508 573448329 399122112662938168
390098456 80151674 192075358161600038
328952025 979190678 642327421233862081
49747398 703696101 533991507220621798
144714653 376315098 274650924876244118
65712805 623078605 405890703380726382
988253747 17137503 293054430443586411

8 592805954 128368694
662296785 775868219 366840864789872824
489006256 212454230 59108069775685328
225382523 426541701 85463197697925681
125458683 386991941 39350592346911240
424467760 389636000 249103228188663375
312630269 531550012 124730940409914881
487770740 265761992 135068667170766264
530165396 260761625 303044725911942118

8 423234981 675675819
957870133 782073482 924851228184311224
183627876 524596143 258122766006135520
657758891 920273625 587246413645794130
427276760 479657112 497280714247848515
526681424 303584375 384365732655421172
531651831 884018801 146597209457192200
29726105 847211002 177909381173782690
577296529 524561511 86275057037730580

9 274636437 658404825
393240676 668788092 544336243247072429
218794654 257823687 205596435945908585
924683044 78553976 285550943905183371
646422780 259056805 334447022721474577
889475857 723813849 582634437235058522
216724631 121499348 113599059900602972
159973423 450265827 245165767224636874
360835442 225630252 234637474781764426
244385162 434959591 314213181030938475

10 603101094 655195686
361280083 126323961 128270454768308201
957019495 630879645 706512073897826544
928118729 688223654 910539267986501071
285183537 801216153 151608256142089554
976137343 225698300 495097549539699426
775516601 780807842 176563048894545034
197668588 413470289 276486289512301836
273767332 385716120 386050267945898048
439937367 73198471 297580263309612478
744126454 856753274 959109141411728101

5 309260068 298150923
856918555 463646901 336037167988667791
336690867 626447663 227747140055168680
32914249 704998334 187688788765117994
831272217 506001845 254809849567400841
605983420 656872394 301247695714972618

6 85753358 453837750
930549204 300624567 210616214519019868
994839333 216476996 169612514761065885
450259074 178955282 114953257390067397
675559562 898028011 419057173432918572
858113697 355378701 224239177116406684
20306910 226533794 101184929629799964

10 450124058 231996244
356421987 726001929 282363702756914472
498585268 375637978 271995979029991011
702196342 890082678 507272245416406617
241407780 650218005 224468418047615767
695960803 352931533 387353236861425957
214799412 57013065 96843683288182862
876649448 425905605 455161042421079469
923154131 518291429 532946106305516073
542462548 7735790 241598811748167211
638601056 93135545 274316141737767473

10 992524271 57825942
709383499 951016241 719479416761343626
243096281 289306810 234004585198688965
695919859 452672602 593705383571321372
41305540 216809037 51402631715663249
527225333 676685890 524150966413248451
229176863 416906117 233359217560019164
162023303 70111471 138431907499410088
767379078 892666712 767445700986955159
299696694 939211306 347263293425035414
270019552 209631601 267246164812377851

8 649508450 898814484
745055483 55599297 517611674781460517
773953461 685847406 746233897356608195
975463166 929677017 934762320167679685
542580475 867303757 920901558994309351
606963592 234197906 381611159463956396
410561520 956280798 647167797081272700
216777217 65618145 155226174438860941
908966591 602931491 951772631447611546

7 946378922 587188845
95068648 465201615 360418807676036720
865331252 955606611 1287212564318206415
879895863 301754802 900822907751121716
102514438 900690316 542684902401841730
56502295 705373648 449147495721776639
821742693 191388770 816817524663896997
677167559 846513558 956583790076767659

10 34260521 284942409
469720921 862884135 173501050825310895
410210960 263506250 80068240280764613
342541400 553683743 124909983609757844
402620073 519412192 114929217776636392
451172562 563678607 143199178005611784
460159782 180890124 40092380279237347
42703140 391517004 87488609695773918
171635738 827211372 231280685376576075
49096781 452270182 80919908986699950
460907150 536389804 98875466159131977

9 558292603 412697820
144303074 193439007 153536620002803833
977513554 810912099 790067107072423601
936749815 667694746 635944642873249124
60726951 924615065 385368621685659315
285413429 215425840 228175270205860270
175401970 60981044 116624640557769520
656981366 316831748 440181086852198138
550222726 51881445 318024194467188569
992105480 721376247 798208568621283793

8 631622690 192515886
755840182 83376826 404335752023944812
682723184 705462096 462759246735323286
286774249 846702929 328020591909832754
333969482 743111061 250365164061159769
679305994 733534517 439287131794245961
159273750 218143792 121777029320971145
740999151 318908792 452658861881751761
136427392 840978992 239091615647689786

5 342899257 813691765
444939473 192585919 221463086931086354
269121507 723016252 46829049427113239
946330083 451217926 68656826228986491
43496365 110078342 26275497921021848
329217769 536735059 169806878393859310

8 373791437 456540397
332492937 790474555 241439396091432014
298327494 299227988 100140284923826453
33365141 428268338 141669135078093405
504106456 662794718 385956804043511509
266184683 785036664 211151034158701997
34273486 78479666 27434920346776818
11800164 983358109 442221276602732561
99796875 85229351 70119499564026470

6 68021769 58636778
890330421 517845111 90026287026636694
571890330 125589368 38597925280839242
368886938 138481623 23461643406646951
982819928 482894797 92075957120014376
767202616 170159807 48476600075526954
102948361 488668336 34022486748425773

5 14704959 230341354
321624014 740741128 97923928329838303
354156432 554226060 92384277199962629
166072298 365537636 61347165408527163
134596296 175476371 11615186770591817
618449433 736635352 137035649928276096

6 330154357 545841651
382761397 926860392 268554081884463098
12378031 489542496 158513402708474948
526895024 361738872 209448469577678292
254565075 18175747 45182438401255624
32391676 972666646 292808692678020017
39677173 666397002 276318715713976931

8 658814352 626725566
368645779 844597351 676921256540750765
497847371 975186911 727691853170798406
355994219 593502058 526579690025522487
244768383 705420470 379075500695015929
780280471 428633396 553663444127183003
132207384 721528985 349281761033608280
128585688 432183361 344093070955830521
990227615 951268878 1047483059726352923

5 273483646 752751215
3879985 880917310 320691245919527050
996699155 928141297 899189646500712247
792978610 79206411 63206443076364537
716282371 22780390 123651781967753204
550472366 270449778 266025378362421709

5 931987664 95886650
13151539 333764154 26875948308727339
849576846 444658754 621431556618094424
158768919 927576054 222068140528303780
674129105 276269492 534164864431559802
955297949 735050307 389583043531598104

5 862164156 976125153
738244126 47642157 537604416713246875
266638155 9723126 195521830298197681
302872754 914373733 286519733119503759
415730301 84537648 332107596692418989
510729290 89708512 422184587461904513

8 401258423 37299205
366162779 626881734 96696513909644951
944708044 744426202 327126383170400966
24350602 224750949 10277962575221816
345207092 878552243 50057171794995058
702395355 483865776 226489967325507402
985225653 588353989 377590079795820555
912243120 243084224 221487185043048627
918179840 288679868 93423743687537647

10 931454118 109463751
142826299 219521206 115134683553352093
900234512 588410667 636602058550069035
401480023 706884399 304194898673554296
203125278 808199446 232510058968360536
733327772 285229686 567206840710184044
298244303 170768632 290900180231300617
956769013 316568435 769735969714135174
425050137 223751274 304799930268365238
151895205 10643775 137086177163540584
10936765 244631647 24989994058816263

10 426302290 642815476
689637122 19841489 156980993975141876
331202410 589746768 488775656912679058
716146369 5222911 202401183648788993
586291798 998507095 576314741549892697
738959979 421877589 370553845145781803
598767698 632040355 570469840139404790
92491615 415038319 269964137977284519
144177166 310018652 148846454636614164
72615825 375199091 203094421808161481
930846641 58062898 190743214771881409

7 895462418 114838665
484182186 710026657 477401026608190951
525780552 478277654 491551699661739729
412287527 510394706 393929153056617745
926771202 464475746 873717840132828553
387060282 210730650 357100968972227941
325089665 580496385 313054728062527622
939918321 993341775 911482316799126358

6 943722934 330045179
880124416 542325208 886633306439046994
279199516 703616783 494984873457651235
978741198 214554422 904814385541390841
974506177 787627191 1088874367830540185
70406617 250927713 113286043866659975
291139500 364404536 317932044374592394

10 208022661 688779764
422540418 561131496 390612344619575182
746214033 386139204 209114553816012344
140714418 595725325 270435894428105963
399063348 72293060 74336311331408308
377682038 858403545 492532973181528452
25993290 234142741 130577060833062154
61401489 194196653 98895672270599065
685649330 128936733 162473444899796619
897832736 123938766 224484898239794621
965154380 54429424 182935933759084660

10 666003841 850848322
996781452 19922520 242071094573887654
113776176 421088231 349808503317259147
673035330 561139794 694864437914172233
366586644 719915477 617966629177268805
703704458 390141476 610578412845689864
931418686 653778807 1129760349941747622
312348368 277028736 229383665792624725
37258565 994933696 868372705632423320
73553563 992532404 777223925952665202
167402572 720139347 562735816835770516

5 499620150 177337498
385903336 847434633 343044540161097741
770050371 447086286 368140464579002926
492335249 243657595 279327726389672935
301285254 248380319 92725882160565077
875666838 357885129 250119351034136913

6 906144096 676785566
184567723 546791998 426034695058174662
273177296 777488639 628187993048099762
911501854 158141574 526278718938014263
469588084 529088656 281777275829570501
608078430 553907676 830820994934371085
560581845 528231577 368115074907356031

7 247035380 240535973
799754245 657063583 304632515958520592
936810321 906819948 394280177536820726
883285318 442794548 275053389409237515
131827053 629348959 156277967946872950
609554680 303147725 220347335621072084
692495620 266408331 145430653947030387
234271821 972962631 222790352016813915

10 298795030 293628144
714855702 929847060 461192666787168501
281270871 684415010 159297036484951568
246502809 396552720 142434339635327788
799964717 566627471 221854151796529304
226342398 847450465 131608569193100513
195338280 595996322 189921777374205144
915117960 970319906 530235653527489668
562551725 244181277 158723037613702314
106864710 800043234 256963135145274535
7562534 134364484 40512744751409618

6 31327032 848965717
845999736 168517081 137641902179533573
487079852 195381223 156267100687492027
306603711 959350878 783745437754060882
666898343 562747648 453175515631731640
115418615 12065924 13319969683711966
763886328 38560869 48599460577536458

10 497839098 651835899
377425648 947228028 635865082931534094
319526530 962271818 416111074417636896
126306596 223111026 197772799227857239
917218960 662854990 621965369250554292
289210036 215439798 238637165709048324
837524673 711490574 546183747828389094
83284718 430562181 164072716535547436
53927814 703431490 218352746184712843
603587600 342015477 506246952623747034
897465561 389805458 688930712894969624

7 263614143 857585455
850262241 750065225 438287775818308756
706981707 141998748 269111877508370894
485796447 782473058 361393978038243530
889731749 655612909 784581909246837872
262793695 116563764 134072567699021587
830521931 989458783 391985740279277323
996177862 92674286 340611237354536145

10 543052380 128722471
697269502 180358607 377110154327371001
572917984 117893528 42328264153533263
142919042 608871492 90622942152670964
244811950 71905540 92653911170419441
115164808 259766536 60325018725461665
946631506 893074711 606299677264418508
554718216 248018041 174772991178411932
614693982 972285064 241198761718201674
501519811 330886636 184048433653128293
840950106 557435213 361793085610939660

8 990253589 152742602
182080763 709916666 226686577693098641
981305292 732042128 447516125150020397
573154392 797230844 623702519948767754
609274065 556707338 203461203355351939
611073937 505881465 525791018262955533
916327029 962867474 242314035824220772
801122189 83922191 787888004277319330
175754651 531698359 199056754704412757

5 714171184 389948029
326182159 82953631 261948359001313133
554753480 197724354 386541900695015010
536081799 755681244 511348520008263892
615267395 246300496 528319743469918231
140610238 791783053 372162030532119673

7 815645916 690960801
259197988 248602025 255288032671566856
486862989 635788378 675913993417366003
136579247 865188574 574228518375735211
455796612 974691662 827742005868764269
545401578 492061236 558275053773905626
857681703 813282016 1231249426818706520
10735042 180103746 125831078033859353

5 9302342 851822673
664656413 111880622 96283377372939605
207695519 765625147 463100437439791903
890401663 953499036 649255741268222786
741054140 76977995 59952619933836672
391238481 295359301 202789575458644208

6 155377200 978248831
768817761 494263232 492037462162914626
977604954 741358032 296266313300426249
506504693 744783404 400860947435762117
907181159 689285911 656560462179502039
924897768 535123030 648858149882349250
804776126 976704074 759348520865961904

8 673606080 547327254
842268622 45586078 481559711455276360
686544597 102925725 465507945277953633
411853126 190625042 375143992570742294
308526954 177834050 303445688356422003
976279020 12805772 611073846608347368
678651033 394208781 574308688519167170
350781491 216879888 289385216350692771
184872796 83343477 163576471757471826

9 761488425 738169301
694237911 319075793 752055257633147374
589249814 986155303 756174607104529008
982514458 212596030 63163948579040328
62225494 722501902 321362991067701939
225814328 184168018 98845637180367811
107698698 749110776 552575239659434680
76660373 389046628 233057641381359065
88858089 284982033 80809988687411987
472076315 435537768 227210125046344843

7 670781247 288702792
468305417 643785044 434881771133438151
845929119 232603871 370212774769068799
25014704 259230136 47031047408232908
735964181 1315173 480067049145852000
677929721 819240121 612061719846791100
533173038 371633850 435036390049818336
806190853 831630193 661805665814126780

5 438027746 283701509
342622931 785203820 287650850273280240
828638783 763856516 522190673005706331
995300828 957643761 436148251450018555
842260608 205256991 108725058494345053
740083688 660612970 233181007682474815

8 28186584 589180099
547362512 96262118 18132272593300441
188149154 155242741 23479820069646049
898031217 591721869 196783066484385859
73885783 208910671 122074780016913335
204925102 941274454 529013818220472347
51141816 847688090 430750037760187457
336135546 786892423 218781856324718222
975709614 767698042 236091370267163866

9 528102584 701813629
540058911 439285981 424911946513825448
149495277 808041910 571186157519556793
955411722 261903789 541291286414403731
46166034 365411327 94473476920632313
171217123 437080893 151708375662412173
611920714 54161658 333642511517023575
440248160 523912943 250792764482124815
10250428 355382885 35897795596911826
622948735 426697510 409233565472662766

7 726723882 334484075
430107828 241541120 314279796970917059
726679979 639047843 635738390058028138
215279225 981647239 473681531725667067
413630177 841949057 390793775715750277
117022031 926904512 354153139150230964
159264605 998192649 441379156742370742
973539375 260911547 674540099957802047

8 120465193 878770238
727665322 961877956 537432986320607443
556559448 821978521 547657791061413439
846818631 963856732 734824922626594580
85368759 252826385 138405835591335245
600143730 15969401 82840106118175967
175406258 104408240 95704020062683229
794149040 68408937 126048912530856173
298323373 904088315 497447327059691098

5 888509674 160833838
802331516 632508793 546772729160868243
202307817 291122897 137547775260715615
643133040 79140839 328025659903490137
444375008 934501206 327342503348021333
609880453 349099365 374112468791597782

9 323665520 593690808
818203934 727783483 631072238439605779
991052744 134775793 347420695325678498
229756550 644352167 384437944135586404
659893185 680850824 567727971550538793
210500795 687670934 396192321206833948
726494445 205987247 301307944575119818
309912654 391471579 308044613572028799
840376413 810397607 694855834200298101
573081496 717796080 453590499409421742

10 692543696 192908776
701008232 924468292 292077725214292680
863435417 263766519 520271782498871280
417885124 597645615 299144066892528691
982288335 869494842 432951410804727538
115971175 595366968 136170776043179431
365399514 996707419 272038360308723770
778437083 13439420 408797779110016273
169976912 111105087 113131796269165962
292940256 621538125 298025878208411544
510375422 365907738 368492456376551465

7 739633878 381234171
632365078 112888308 244354496637268585
653144678 130674081 135128552799640097
359477877 519843130 339851032819500765
121471976 997386216 260981168287059591
342121307 201341651 150399363875201352
524579414 386891079 335812488732585698
12049811 311571272 70106525170078372
