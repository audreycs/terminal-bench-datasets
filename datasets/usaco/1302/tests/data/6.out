254731047
811564290
682045443
616343738
729788410
193178213
271547955
446695683
649028647
377611589
397427695
179180554
357139677
415707354
681761610
965874322
974894164
661310204
473391288
369207976
36994220
426391692
303137949
294072786
618827447
198828954
275206158
631130563
401973689
136107595
463772330
838609939
173553678
338003205
707647737
184788437
446359421
288600158
184152034
702708081
401794950
417068397
245228358
417511098
861978180
502098030
559161873
934439990
214010140
1030625253
199425434
52524899
89713545
177014904
620142597
228138878
131751585
179951745
198858451
1060019281
494659505
26433072
175428242
491405838
413848851
619019018
604736335
1041111740
323513206
317956001
492604501
77027662
222583058
403401707
695474425
338049887
985891995
156577678
285872094
80486617
455616156
682693468
495658655
878555628
250245310
497208537
249480412
733999535
187032680
1213932907
418171923
378077964
466120835
666890702
266827208
411696602
438937126
225587134
468798699
739033611
